packages/dd-trace/src/opentracing/propagation/text_map.js

```diff
@@ -60,6 +70,7 @@ class TextMapPropagator {
     this._injectB3MultipleHeaders(spanContext, carrier)
     this._injectB3SingleHeader(spanContext, carrier)
     this._injectTraceparent(spanContext, carrier)
+    this._injectAwsXrayContext(spanContext, carrier)
```

Comment:
I'm admittedly not too familiar with the opentracing/ directory. Do your changes result in the tracer automatically injecting AWS xray tracing headers into signed outbound requests? I would have expected to see changes made in `packages/datadog-plugin-http/src/client.js`.

Reply:
So this file contains all the propagation styles implemented via inject / extract methods. By default, I think we only use the Datadog and Trace Context propagators, but there's also B3, baggage, and now AWS X-Ray. Users can configure which HTTP propagators they want, and when `xray` is included in that list, we will use the code I added. `packages/datadog-plugin-http/src/client.js` calls these methods via `this.tracer.inject|extract()`.

Reply:
That makes sense. Your solution is to add is an additional context propagation mode instead of changing request headers at the last moment.

Have you confirmed that when two modes are present that both sets of headers are used simultaneously? Reading `_extractSpanContext()` suggests that it works this way (by breaking out of the switch and not the for loop) but can you add a test to be sure? E.g. set the tracing types to `datadog,xray` and confirm that the two tracing headers are present?